conan/tools/scm/git.py

```diff
@@ -120,15 +120,16 @@ def commit_in_remote(self, commit, remote="origin"):
             # Don't raise an error because the fetch could fail for many more reasons than the branch.
             return False
 
-    def is_dirty(self):
+    def is_dirty(self, repository=False):
         """
         Returns if the current folder is dirty, running ``git status -s``
         The ``Git(..., excluded=[])`` argument and the ``core.scm:excluded`` configuration will
         define file patterns to be skipped from this check.
 
```

Comment:
Shouldnt we document the repository argument in the docstring?

Reply:
yes, sorry I missed that, adding it.